### PR TITLE
Feature/e UI 4753 service hearing value

### PR DESCRIPTION
**Pull request by rayliangatos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4753

### Change description ###
Service API and ngrx store

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Kartheek-S — APPROVED** [review]

File: src/hearings/store/effects/hearing-values.effects.spec.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        },
+        HearingValuesEffects,
+        provideMockActions(() => actions$)
+      ]
+    });
+    effects = TestBed.get(HearingValuesEffects);
+  });
+
+  describe('loadHearingValue$', () => {
+    const SERVICE_HEARING_VALUES: ServiceHearingValuesModel = {
```
[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/store/effects/hearing-values.effects.spec.ts:34]: Do you mean SERVICE_HEARING_VALUES?
In current naming convention we define constants with UPPER_SNAKE_CASE.

File: src/hearings/store/reducers/hearing-values.reducer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import {RequirementType} from '../../models/hearings.enum';
+import {ServiceHearingValuesModel} from '../../models/serviceHearingValues.model';
+import * as fromHearingValuesActions from '../actions/hearing-values.action';
+import * as fromHearingValuesReducer from './hearing-values.reducer';
+
+describe('Hearing Values Reducer', () => {
+
```
[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/store/reducers/hearing-values.reducer.spec.ts]: I have removed it though I think it is ok

File: src/hearings/store/reducers/hearing-values.reducer.spec.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  describe('Actions', () => {
+
```
[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/store/reducers/hearing-values.reducer.spec.ts]: I have removed it though I think it is ok

File: src/hearings/store/selectors/hearing-values.selectors.spec.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        StoreModule.forFeature('hearings', reducers),
+      ],
+    });
+    store = TestBed.get(Store);
+    spyOn(store, 'dispatch').and.callThrough();
+  });
+
+  describe('getHearingValues', () => {
+    it('should return hearing values state', () => {
+      let result;
```
[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/store/selectors/hearing-values.selectors.spec.ts]: done

File: src/hearings/store/effects/hearing-values.effects.spec.ts [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        flagAmendURL: '/',
+      },
+      screenFlow: [
+      ],
+      vocabulary: [
+        {
+          word1: '',
+        },
+      ],
+    };
```
[the remark this answers is not in the document]
Comment (rayliangatos) [reply src/hearings/store/effects/hearing-values.effects.spec.ts:119]: done

**Review by waynepHMCTS — APPROVED** [review]

**Pull request merged**